inc/Classes/Validator/Email.php

```diff
@@ -139,9 +139,6 @@ public function validate($value)
 
     /**
      * Check DNS Records for MX type.
-     *
-     * @param string $host
```

Comment:
Do you mean this line or the overall change of replacing PHP doc hints with actual PHP types?

This one is getting removed because the function signature already looks like

```php
private function checkMX(string $host): bool
```

The PHP doc here is duplicate information.
Modern API generators and IDEs supporting PHP types gather this data rather than from the function signature. Primarily because this is the code that matters + is checked during runtime.

Reply:
> Do you mean this line or the overall change of replacing PHP doc hints with actual PHP types?

Yes. ;)
I was raising the question in general, because I noticed it in this instance.
But I understand it was intentional, which was the first part of the question.

phpDoc is duplicate information as long as nobody writes a proper function/parameter documentation.
But I guess we should do that as some point, shouldn't we? Or is there a better/preferred way for that?

the other part was that I was wondering if phpDoc is able to use php types or if it still relies on these tags and thus removing it will also remove the related sections in the documentation.

Reply:
> But I guess we should do that as some point, shouldn't we? Or is there a better/preferred way for that?

We should do this at some point.
Having a quick chat about what proper docs look like is beneficial.

> the other part was that I was wondering if phpDoc is able to use php types or if it still relies on these tags and thus removing it will also remove the related sections in the documentation.

We should, once the most PRs are merged.
IMO we should work on this once we package a new release.